### DISCUSSION TITLE
Special build for Android 6.0

### DISCUSSION THREAD

**Discussion by Rikk** [Alex313031/Thorium-Special#50] [General]
Hello @Alex313031. Your approach to support Win7/8.1 with branch 109 is very nice and appreciated. While it was immensely easier for me to recreate (with Win10) the old installation of Windows in my notebook, the same is not true about my phone running Android 6, which is an OS tied to hardware in a way much deeper than Windows is.

Considering you are making an effort to maintain Thorium 109 up-to-date, and (from what I understand) branch 109 is also the last that can run on Android 6.0 (api 23), I'd like to ask if it would be possible for you to release a special build for Android 6 using that Chromium version, so we that are stuck on outdated Android can have a nice browser with the most up-to-date Chromium that Android 6 will ever support, just like Win7/8.1.

May I ask a side question: is the Android version of Thorium capable of running user-scripts? (I use Kiwi for that, but I think Bromite can too)

Thank you!

**Comment by gz83** [comment]
@Rikk 

The thorium for win7/8/8.1 system has been updated recently, as to whether the Android version will be updated, still need @Alex313031  to make the final decision

In addition, the Android version of thorium currently cannot install extensions or scripts for the time being.
